testsuite/features/support/env.rb

```diff
@@ -332,6 +328,11 @@ def enable_assertions
   skip_this_scenario unless $service_pack_migration_enabled
 end
 
+# do test only if we don't want to run service pack migration
+Before('@skip_sp_migration') do
+  skip_this_scenario if $service_pack_migration_enabled
```

Comment:
hmmm 🤔 the name is not coherent right?
If this variable is enabled then we DON'T want to skip it. If it's false, then we want to skip the scenario.

Reply:
To me skip is very clear :-)